src/clients/MultiCallerClient.ts

```diff
@@ -354,7 +359,7 @@ export class MultiCallerClient {
     return {
       chainId,
       contract,
-      method: "multicall",
+      method: multicallMethod,
```

Comment:
If you use `method` instead of `multicallMethod` then you can drop the assignment.

```suggestion
      method,
```